prow/plank/reconciler.go

```diff
@@ -146,23 +146,28 @@ func newReconciler(ctx context.Context, pjClient ctrlruntimeclient.Client, overw
 		opener:             opener,
 		totURL:             totURL,
 		clock:              clock.RealClock{},
-		serializationLocks: &shardedLock{
+		jobSerializationLocks: &shardedLock{
+			mapLock: &sync.Mutex{},
+			locks:   map[string]*semaphore.Weighted{},
+		},
+		queueSerializationLocks: &shardedLock{
 			mapLock: &sync.Mutex{},
 			locks:   map[string]*semaphore.Weighted{},
 		},
 	}
 }
 
 type reconciler struct {
-	pjClient           ctrlruntimeclient.Client
-	buildClients       map[string]ctrlruntimeclient.Client
-	overwriteReconcile reconcile.Func
-	log                *logrus.Entry
-	config             config.Getter
-	opener             io.Opener
-	totURL             string
-	clock              clock.WithTickerAndDelayedExecution
-	serializationLocks *shardedLock
+	pjClient                ctrlruntimeclient.Client
+	buildClients            map[string]ctrlruntimeclient.Client
+	overwriteReconcile      reconcile.Func
+	log                     *logrus.Entry
+	config                  config.Getter
+	opener                  io.Opener
+	totURL                  string
+	clock                   clock.WithTickerAndDelayedExecution
+	jobSerializationLocks   *shardedLock
```

Comment:
I'd suggest "concurrencySerializationLocks" to help associate it with pj.spec.MaxConcurrency. (Whereas queueSerializationLocks is for pj.spec.JobQueueName which uses "job" and makes this a bit confusing.)

Reply:
If we're going that route (I don't mind the current naming), it probably doesn't hurt to just say the full "maxConcurrencySerializationLocks" instead. I don't think "concurrencySerializationLocks" is much more readable on its own because there are multiple kinds of concurrency happening in this area.

Reply:
Yeah, maxConcurrencySerializationLocks seems fine. Also, "jobQueueSerializationLocks" would be consistent, and no confusion would arise.

Reply:
Renamed.